src/pages/home/HeaderView.js

```diff
@@ -158,7 +158,7 @@ const HeaderView = (props) => {
                             {props.report.hasOutstandingIOU && (
                                 <IOUBadge iouReportID={props.report.iouReportID} />
                             )}
-                            <VideoChatButtonAndMenu isConcierge={isConcierge} />
+                            <VideoChatButtonAndMenu openInboxCall={isConcierge} taskID="ExpensifyCashConciergeDM" />
```

Comment:
Thoughts on making the 2nd param consistent and more verbose i.e., instead of `taskID` calling it `inboxCallTaskID`.
Two reasons why I suggest this,
1. `taskID` for `VideoChatButtonAndMenu` doesn't mean much imo so someone needs to look at the code to know what it is.
2. Renaming to `inboxCallTaskID` would keep it consistent to the variable [here](https://github.com/Expensify/App/pull/4637/files#diff-ddaba852a06cdf38ad4b8bd91d4c2bf0a7055b1e18016f5ee2f058f90c71b986R114).